apps/web/components/shared/contactUs/ContactUs.types.ts

```diff
@@ -17,7 +17,16 @@ export type ContactUsFormProps = PropsWithClassName & {
     query: FormInitialValue;
     role: FormInitialValue;
   };
+  hiddenFields?: HiddenField;
   buttonText: string;
+  formSenderFn?: Function;
+};
+
+export type HiddenField = {
+  [key: string]: {
```

Comment:
```suggestion
  [key: 'name' | 'lastname' | 'email' | 'role' | 'query' | 'phone']: {
```
In this way, then you can use dot notation in the form.